src/authorizer.js

```diff
@@ -115,7 +115,7 @@ class Authorizer {
     try {
       identity = await jwtManager.verify(token, key, { algorithms: ['RS256'] });
     } catch (exception) {
-      this.logFunction({ level: 'WARN', title: 'Unauthorized', details: 'Error verifying token', error: exception, method: methodArn });
+      this.logFunction({ level: 'WARN', title: 'Unauthorized', details: 'Error verifying token', error: exception, method: methodArn, token });
```

Comment:
That makes sense, should we add it to line 103 as well?

Reply:
Makes sense. Done.